client/services/code_preprocessors/PythonCodePreprocessorService.js

```diff
@@ -165,9 +165,13 @@ tie.factory('PythonCodePreprocessorService', [
         VARNAME_CORRECTNESS_TEST_RESULTS + ' = []',
       ].join('\n');
       for (var i = 0; i < correctnessTests.length; i++) {
+        var testInputCode = (
+          inputFunctionName ? inputFunctionName + '(' + VARNAME_TEST_INPUTS + 
```

Comment:
Another way might be:

    var inputVarCode = VARNAME_TEST_INPUTS + '[' + i + ']';
    var testInputCode = inputFunctionName ? inputFunctionName + '(' + inputVarCode + ')' : inputVarCode;

And yet another way might be to define

    var generateFunctionCallCode = function(functionName, arg) {
      return functionName + '(' + arg + ')';
    };

    var testInputCode = inputFunctionName ? generateFunctionCallCode(inputFunctionName, inputVarCode) : inputVarCode;

I think these might be cleaner.